quinn-udp/src/windows.rs

```diff
@@ -470,3 +493,28 @@ static MAX_GSO_SEGMENTS: Lazy<usize> = Lazy::new(|| {
         Err(_) => 1,
     }
 });
+
+/// Evaluates to [`Ok(true)`] if executed either directly on Windows on ARM, or
+/// on an emulator which itself executes on Windows on ARM.
+///
+/// See
+/// <https://learn.microsoft.com/en-us/windows/arm/apps-on-arm-x86-emulation#detecting-emulation>
+/// for details.
+static IS_WINDOWS_ON_ARM: LazyLock<io::Result<bool>> = LazyLock::new(|| {
```

Comment:
Still not sure what is going on with MSRV CI, but still pretty sure `LazyLock` doesn't work with our stated MSRV.

Reply:
Thank you! Moved to `once_cell::sync::Lazy` as we already do for `MAX_GSO_SEGMENTS`.